dpkgfiles/make_deb_package.README

```diff
@@ -0,0 +1,48 @@
+Instructions for using make_deb_package.py outside of Gitian
+
+
+1) We use cowbuilder to build in a chroot, so you need to prepare a chroot with
```

Comment:
At some point, the documentation needs to reflect the fact that, due to various tool requirements, utopic is the minimum required if using an Ubuntu distro. (I don't know about Debian. Just start somewhere and go back and forth as necessary!) This is separate from the requirements to build Armory altogether. I don't think you need to mention that, though, as long as you clarify that the requirement is due to autotools requirements and such.

(Hmmm. Come to think of it, autotools is going to make 14.04 users unhappy. We'll have to let them know that they'll need to go through the Debian process as a workaround.)